iati/core/data.py

```diff
@@ -154,6 +154,36 @@ def _raw_source_at_line(self, line_number):
         except IndexError:
             raise ValueError
 
+    @property
+    def version(self):
+        """Finds the version that this Dataset is specified against.
```

Comment:
Because this is a property rather than a function, the docstring should detail what it _is_ rather than what is _does_.